ui/src/tempVars/components/TagKeysTemplateBuilder.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {PureComponent} from 'react'
+
+import KeysTemplateBuilder from 'src/tempVars/components/KeysTemplateBuilder'
+import {TemplateBuilderProps, TemplateValueType} from 'src/types'
+import {proxy} from 'src/utils/queryUrlGenerator'
+import parseShowTagKeys from 'src/shared/parsing/showTagKeys'
+
+export const fetchTagKeys = async (
```

Comment:
1. this kind of seems like a utility function that i might expect outside the component.. thoughts?
1. could you type the inputs?